internal/updater/updater.go

```diff
@@ -126,6 +136,8 @@ func (u *AvailableUpdate) InstallBlocking(installTargetPath string, args ...stri
 	if u.Tag != nil {
 		envs = append(envs, fmt.Sprintf("%s=%s", constants.UpdateTagEnvVarName, *u.Tag))
 	}
+
+	logging.Debug("Executing: %s %s env: %s\n", installerPath, args, envs)
```

Comment:
We're already debug logging this: https://github.com/ActiveState/cli/blob/3bf100013b38a2752b32c98192dde0dc40953860/internal/exeutils/exeutils.go#L120